WcaOnRails/app/models/competition.rb

```diff
@@ -116,10 +117,22 @@ def warnings
         warnings[:name] = "The competition name is longer than 32 characters. We prefer shorter ones and we will be glad if you change it."
       end
     end
+
+    if user && user.can_edit_competition_report?(self) && is_over? && !delegate_report.posted?
+      path_to_edit_report = Rails.application.routes.url_helpers.competition_report_edit_path(self)
+      link_to_edit_report = ActionController::Base.helpers.link_to('here', path_to_edit_report)
+      warnings[:report_not_submitted] = (
+        "".html_safe +
+        "Your report is not posted yet! Click " +
+        link_to_edit_report +
+        " to work on it."
+      )
```

Comment:
Hopefully we can make use of [this](https://github.com/cubing/worldcubeassociation.org/blob/master/WcaOnRails/config/initializers/monkey_patches.rb#L42-L49), right?